src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/BrotliStream.cs

```diff
@@ -72,6 +82,13 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        /// <summary>Asynchronously releases the unmanaged resources used by the <see cref="System.IO.Compression.BrotliStream" />.</summary>
+        /// <returns>A task that represents the asynchronous dispose operation.</returns>
+        /// <remarks><![CDATA[
+        /// The `DisposeAsync` method lets you perform a resource-intensive dispose operation without blocking the main thread. This performance consideration is particularly important in a [!INCLUDE[win8_appname_long](~/includes/win8-appname-long-md.md)] app or [!INCLUDE[desktop_appname](~/includes/desktop-appname-md.md)] app where a time-consuming stream operation can block the UI thread and make your app appear as if it is not working. The async methods are used in conjunction with the `async` and `await` keywords in Visual Basic and C#.
+        /// This method disposes the Brotli stream by writing any changes to the backing store and closing the stream to release resources.
+        /// Calling `DisposeAsync` allows the resources used by the <xref:System.IO.Compression.BrotliStream> to be reallocated for other purposes. For more information, see [Cleaning Up Unmanaged Resources](/dotnet/standard/garbage-collection/unmanaged).
```

Comment:
`xref` links could be converted to `<see cref="">` but then we probably have to find out how many other tags are worth converting.

```suggestion
        /// Calling `DisposeAsync` allows the resources used by the <see cref="System.IO.Compression.BrotliStream"> to be reallocated for other purposes. For more information, see [Cleaning Up Unmanaged Resources](/dotnet/standard/garbage-collection/unmanaged).
```

Reply:
If we convert xrefs to crefs, there are two problems to tackle:

1. We need to add the API type prefix (`T:` for types, `P:` for properties, `F:` for vields, `E:` for events, `M:` for methods, `N:` for namespaces). I need to check if Roslyn has a way to obtain the full DocID with this prefix.
2. We would lose the suffix `?displayProperty=nameWithType`, which is invalid in crefs.

As we discussed earlier, we may be able to remove the `CDATA` tags and leave the `xref` items in remarks unmodified, without causing a build failure due to unresolved API full names. The triple slash comments seem to treat `xref` items as plain text, not as xml items. CC @danmosemsft

Reply:
I like that you're doing just this one library so we can iterate a bit and find an approach that works really well (with respect to CDATA, line breaks, INCLUDE, <xref> and anything else) and that devs like before extending to other libraries.

(BTW as an aside we apparently have an analyzer that flags "T:" etc in our build -- I know because I had to remove them when porting System.Speech. We'd have to disable that.)

Reply:
Presumably aspnetcore must have solved (or at least made decisions about) each of these things already?

Reply:
> We need to add the API type prefix (T: for types, P: for properties, F: for vields, E: for events, M: for methods, N: for namespaces). I need to check if Roslyn has a way to obtain the full DocID with this prefix.

Why?  Roslyn emits those into the underlying XML automatically based on resolving the name.  We should strongly prefer using the language-supported syntax.

> We would lose the suffix ?displayProperty=nameWithType, which is invalid in crefs.

Why is that necessary?  My preference is that we only force the different syntax when there's something critical it expresses that can't be done with the existing, well-supported / recognized-by-IntelliSense / etc. syntax.

Reply:
Thanks for correcting me, @stephentoub. The prefixes are not needed, which would make it easier for remarks to be transformed to `<see cref="">` items in triple slash.

I forgot to mention, but I also have to detect remarks items that are wrapped by backticks, which point to paramrefs or typeparamrefs. I would have to find out what it's referring to, and if nothing is found, then I have to transform it into a `<see langword="">`.

Reply:
I did an experiment in MS Docs to verify point no. 5, and I was wrong: sentences are not handled as independent paragraphs by the MS Docs CSS. See the test PR results: https://github.com/dotnet/dotnet-api-docs/pull/5213/files

So if we want to preserve paragraphs, we will have to keep empty lines, which would make several remarks quite large.

Reply:
> So if we want to preserve paragraphs, we will have to keep empty lines, which would make several remarks quite large.

Preservation of paragraph boundaries because that's how they currently are isn't required.  Preservation of desired paragraph boundaries is desirable.  Such boundaries generally are valuable in large part of help with reading comprehension, which makes them valuable in comments as well.